eclass/cron.eclass

```diff
@@ -17,18 +18,35 @@
 # chosen based on the most common setting among cron ebuilds.
 #
 
-inherit eutils flag-o-matic
+case ${EAPI:-0} in
+	[0-5]) die "cron.eclass: EAPI ${EAPI} is too old." ;;
+	[67]) ;;
+	*) die "EAPI ${EAPI} is not supported by cron.eclass." ;;
+esac
```

Comment:
Looking at it again, this is no longer the recommended style. It should look like this:
```suggestion
case ${EAPI} in
	6|7) ;;
	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
esac
```
Same for the other eclasses.